web/packages/teleport/src/services/cta/cta.ts

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright 2023 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// ctaService dictates if some CTAs should be hidden or visible in the app.
+// Currently, no CTAs are active.
+export const ctaService = {
+  showAuthConnectorsCTA: false,
+  showTrustedDevicesCTA: false,
+  showAccessRequestsCTA: false,
+  showSupportCTA: false,
+  showActiveSessionsCTA: false,
+};
```

Comment:
Yeah, since these will depend on things like cluster features and the customer plan, I was thinking of moving this to the teleport context (`web/packages/teleport/src/teleportContext.ts`) and making it a method there like `getFeatureFlags()`. How does that sound?

Reply:
Cool, updated in https://github.com/gravitational/teleport/pull/25073/commits/bcb9b4abbc538b335ca668fceadc8baaa5072ead.

Since I'm not sure this will have to be a function, I just added it as an object for now. I'll change if necessary when actually implementing the CTA flags.